OpenRobertaServer/src/main/java/de/fhg/iais/roberta/javaServer/restServices/all/controller/ClientInit.java

```diff
@@ -64,11 +64,15 @@ public ClientInit(
         this.ipToCountry = ipToCountry;
     }
 
+    //////////////////////////////////////////
```

Comment:
If comments are added for clarity they should be in English, for personal use they should be removed in the final version.